src/index.html

```diff
@@ -19,8 +19,8 @@ <h3>Recent scores</h3>
         <section id="add-score">
             <h3>Add your score</h3>
             <form action="submit">
-                <input type="text" name="name" id="input-name">
-                <input type="number" name="score" id="input-score">
+                <input type="text" name="name" id="input-name" placeholder="Your name">
+                <input type="number" name="score" id="input-score" placeholder="Your score">
```

Comment:
[OPTIONAL] When the user tries to submit a score without a score, the API returns an error because the score need's to be a valid value or present to persist the changes on the database. I think it could be great if you further validate this form so the user cannot submit it without a score.

![image](https://user-images.githubusercontent.com/79114233/146302076-5b428756-6bcb-4339-90f3-e2b14819000d.png)

Reply:
it's solved